continuum/task_set.py

```diff
@@ -92,14 +126,21 @@ def get_sample(self, index: int) -> np.ndarray:
     def __getitem__(self, index: int) -> Tuple[np.ndarray, int, int]:
         """Method used by PyTorch's DataLoaders to query a sample and its target."""
         img = self.get_sample(index)
-        y = self.y[index]
-        t = self.t[index]
+        y = self._y[index]
+        t = self._t[index]
 
         if self.trsf is not None:
             img = self.trsf(img)
 
         return img, y, t
 
+    def get_image(self, index):
```

Comment:
Do we need this method? We can simply do `task_set[index][0]` for the same result.

Reply:
seems like a valid point ^^

Reply:
[patch_pr29 7626f16]